lib/rspec/rails/view_rendering.rb

```diff
@@ -44,6 +44,13 @@ def render_views?
       #
       # @private
       class EmptyTemplateResolver < ::ActionView::FileSystemResolver
+
+        def initialize(path, pattern=nil)
+          unless path.is_a?(::ActionView::Resolver)
+            super
+          end
+        end
+
```

Comment:
Doesn't this just end up adding an uninitialised `EmptyTemplateResolver` to the view paths array?